### ISSUE TITLE
Move to Ocamlformat?

### ISSUE THREAD

**Issue by Sir4ur0n** [c-cube/qcheck#112]
As formatting is a matter of taste, I am opening this issue because I massively prefer Ocamlformat over ocp-indent (which - I think ? - QCheck code currently uses) :grin: 

What do you think about moving to Ocamlformat? I find the formatting much more consistent and helpful to read code

**Comment by c-cube** [comment]
If we find an ok style, why not :)

**Comment by vch9** [comment]
I strongly agree with @Sir4ur0n. It also makes win a LOT of time, got pretty sad when I disabled it in this project

**Comment by c-cube** [comment]
I'm ok with it if we can find a stable config that does *not* put `;` first in records. Should be done in between PRs though, because it makes merging/rebasing a nightmare.
